src/chi/components/tables/tables.scss

```diff
@@ -2,8 +2,233 @@
+      height: 3rem;
+    }
+
+    th {
+      border-bottom: 0.0625rem solid set-color(grey, 30);
+      font-weight: 600;
+      height: 0;
+      padding: 1rem;
+      text-transform: capitalize;
+
+      &.-sorted {
+        div {
+          align-items: center;
+          cursor: pointer;
+          display: flex;
+          flex-direction: row;
+
+          &::after {
+            background-image: url('./icon-chevron-thin-up.svg');
+            content: '';
+            display: block;
+            height: 0.75rem;
+            margin-left: 0.5rem;
+            opacity: 0;
+            width: 0.75rem;
+          }
+        }
+
+        & i {
+          margin-left: 0.5rem
+        }
+      }
+
+      &.-ascending {
+        div::after {
+          opacity: 1;
+        }
+      }
+
+      &.-descending {
+        div::after {
+          opacity: 1;
+          transform: rotate(180deg);
+        }
+      }
+    }
+
+    & td {
+      border-bottom: 0.0625rem solid set-color(grey, 20);
+      padding: 1rem;
+      padding-bottom: 0.9375rem;
+      vertical-align: middle;
+    }
+
+    & tbody {
+      tr {
+        &.-row--success {
+          background-color: set-color(green, 10);
+        }
+
+        &.-row--warning {
+          background-color: set-color(yellow, 10);
+        }
+
+        &.-row--danger {
+          background-color: set-color(red, 10);
+        }
+
+        &.-row--info {
+          background-color: set-color(blue, 10);
+        }
+
+        &[disabled] {
+          color: rgba(set-color(grey, 90), 0.4);
+        }
+      }
+    }
+
+    &.-hover {
+      tbody {
+        tr {
+          &:hover,
+          &.-hover {
+            background-color: set-color(grey, 10);
+          }
+        }
+      }
+    }
+
+    &.-striped {
+      tbody {
+        tr:nth-child(odd) {
+          background-color: set-color(grey, 10);
+        }
+      }
+    }
+
+    &.-borderless {
+      &::before {
+        border: 0;
+      }
+
+      th {
+        border-bottom: 0.0625rem solid set-color(grey, 30);
+      }
+
+      td {
+        border: 0;
+        padding: 1rem;
+      }
+    }
+
+    &.-bordered {
+      &::before {
+        border: 0.0625rem solid set-color(grey, 30);
+        border-bottom-width: 0;
+        border-left-width: 0;
+      }
+
+      th {
+        border-bottom: 0;
+      }
+
+      th {
+        border: 0.0625rem solid set-color(grey, 30);
+        border-right-width: 0;
+        border-style: solid;
+
+        &:last-of-type {
+          border-right-width: 0.0625rem;
+        }
+      }
+
+      td {
+        border: 0.0625rem solid set-color(grey, 30);
+        border-right-width: 0;
+        border-top-width: 0;
+
+        &:last-of-type {
+          border-right-width: 0.0625rem;
+        }
+      }
+    }
+
+    &.-small {
+      tr {
+        height: 2rem;
+      }
+
+      th {
+        padding: .5rem 1rem .4375rem .9375rem
+      }
+
+      td {
+        padding-bottom: 0.4375rem;
```

Comment:
@davidojedacl I think it looks aligned vertically to bottom. Can we make it centered?

Reply:
@davidojedacl @jllr only thing that worries me about the current sizes is they're 1.3x larger and 1.5x smaller than the base. if we need to introduce something in between in the future, we'd have to change all the sizes. something like this would be much more scalable:

xl: 64px row height
lg: 56px row height
md(base): 48px row height
sm: 40px row height
xs: 32px row height

Also, right now we are applying fixed heights on the tr and some extra padding-bottom on the td to achieve an even height. Let's remove that and just apply standard padding. The total height for base rows will be 48 + 1px border so 49 total. That's ok.